povm_toolbox/post_processor/dual_from_state.py

```diff
@@ -0,0 +1,49 @@
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""TODO."""
+
+from __future__ import annotations
+
+from typing import cast
+
+import numpy as np
+from qiskit.quantum_info import DensityMatrix, SparsePauliOp, Statevector
+
+from povm_toolbox.quantum_info import BaseDUAL, BasePOVM, MultiQubitPOVM
+from povm_toolbox.quantum_info.multi_qubit_dual import MultiQubitDUAL
+
+
+def optimal_dual_from_state(
```

Comment:
Why did you rename this to `optimize_dual_from_state` rather than just `dual_from_state`?
There is no "optimization" performed here as this is a one-shot computation of the dual from the provided state.

Reply:
It's `optimal` (not optimized) because we have the analytical formula for the optimal dual frame if we know the state. But I agree, maybe we should just align with the other names. Let me know what you think!

Reply:
Oh I totally misread that.. :sweat_smile: 

But yeh, let's keep it as `dual_from_state`. You can document that this is the optimal one (plus, users will likely notice when they realize that they need to input the state into this..)